core-web/libs/template-builder/src/lib/components/template-builder/template-builder.component.ts

```diff
@@ -112,13 +113,14 @@ export class TemplateBuilderComponent implements OnInit, AfterViewInit, OnDestro
         private dialogService: DialogService,
         private dotMessage: DotMessageService
     ) {
-        this.items$ = this.store.items$.pipe(
-            startWith([]),
-            map((items) => parseFromGridStackToDotObject(items))
-        );
+        this.items$ = this.store.items$.pipe(map((items) => parseFromGridStackToDotObject(items)));
 
-        combineLatest([this.items$, this.store.layoutProperties$.pipe(startWith())])
+        combineLatest([this.items$, this.store.layoutProperties$])
             .pipe(
+                startWith([]),
+                filter(([items, layoutProperties]) => !!items && !!layoutProperties),
+                skip(1), // Skip the init of the store
```

Comment:
Can we write a test to ensure we are skipping the `first` value when initiating the `store`?

Reply:
Of course, nice catch!